src/Microsoft.DotNet.Web.ProjectTemplates/content/WebApi-CSharp/Controllers/ValuesController.cs

```diff
@@ -13,7 +13,7 @@ namespace Company.WebApplication1.Controllers
     [Authorize]
 #endif
     [Route("api/[controller]")]
-    public class ValuesController : Controller
+    public class ValuesController : ControllerBase
```

Comment:
Need to annotate this type with `ApiController`

Reply:
Is that all we are doing for this PR then? Derive from ControllerBase and annotate with ApiController? Since FromRoute isn't required.

Reply:
Essentially given ValuesController isn't very complicated.